apps/editing-toolkit/editing-toolkit-plugin/help-center/class-help-center.php

```diff
@@ -124,39 +124,27 @@ public function enqueue_script() {
 			'before'
 		);
 
-		$user_id         = get_current_user_id();
-		$user            = get_userdata( $user_id );
-		$primary_site_id = $user->primary_blog;
-		$user_email      = $user->user_email;
-		$avatar_url      = function_exists( 'wpcom_get_avatar_url' ) ? wpcom_get_avatar_url( $user_email, 64, '', true )[0] : null;
+		$current_user = wp_get_current_user();
+		$user_id      = $current_user->data->ID;
+		$username     = $current_user->data->user_login;
+		$user_email   = $current_user->data->user_email;
 
 		wp_add_inline_script(
 			'help-center-script',
 			'const helpCenterData = ' . wp_json_encode(
 				array(
-					'currentSite'   => $this->get_current_site(),
-					'locale'        => get_locale(),
-					'currentUserId' => get_current_user_id(),
-					'primarySiteId' => $primary_site_id,
-					'userEmail'     => $user_email,
-					'avatarUrl'     => $avatar_url,
-					'displayName'   => $user->display_name,
+					'currentUser' => array(
+						'ID'       => $user_id,
+						'username' => $username,
+						'email'    => $user_email,
```

Comment:
We should add `display_name`, `avatar_URL` for Wapuu. `get_userdata` should give you this info.